subworkflows/local/bam_variant_calling_germline_all/main.nf

```diff
@@ -13,229 +13,201 @@ include { BAM_VARIANT_CALLING_SINGLE_TIDDIT   } from '../bam_variant_calling_sin
-        dbsnp_tbi                         // channel: [mandatory] dbsnp_tbi
-        dict                              // channel: [mandatory] dict
-        fasta                             // channel: [mandatory] fasta
-        fasta_fai                         // channel: [mandatory] fasta_fai
-        intervals                         // channel: [mandatory] intervals/target regions
-        intervals_bed_gz_tbi              // channel: [mandatory] intervals/target regions index zipped and indexed
-        intervals_bed_combined            // channel: [mandatory] intervals/target regions in one file unzipped
-        intervals_bed_combined_haplotypec // channel: [mandatory] intervals/target regions in one file unzipped, no_intervals.bed if no_intervals
-        known_sites_indels
-        known_sites_indels_tbi
-        known_sites_snps
-        known_sites_snps_tbi
+    tools                             // Mandatory, list of tools to apply
+    skip_tools                        // Mandatory, list of tools to skip
+    cram                              // channel: [mandatory] cram
+    bwa                               // channel: [mandatory] bwa
+    dbsnp                             // channel: [mandatory] dbsnp
+    dbsnp_tbi                         // channel: [mandatory] dbsnp_tbi
+    dbsnp_vqsr
+    dict                              // channel: [mandatory] dict
+    fasta                             // channel: [mandatory] fasta
+    fasta_fai                         // channel: [mandatory] fasta_fai
+    intervals                         // channel: [mandatory] intervals/target regions
+    intervals_bed_combined            // channel: [mandatory] intervals/target regions in one file unzipped
+    intervals_bed_combined_haplotypec // channel: [mandatory] intervals/target regions in one file unzipped, no_intervals.bed if no_intervals
+    intervals_bed_gz_tbi              // channel: [mandatory] intervals/target regions index zipped and indexed
+    known_indels_vqsr
+    known_sites_indels
+    known_sites_indels_tbi
+    known_sites_snps
+    known_sites_snps_tbi
+    known_snps_vqsr
 
     main:
-
-    ch_versions         = Channel.empty()
+    versions = Channel.empty()
 
     //TODO: Temporary until the if's can be removed and printing to terminal is prevented with "when" in the modules.config
-    deepvariant_vcf     = Channel.empty()
-    freebayes_vcf       = Channel.empty()
-    haplotypecaller_vcf = Channel.empty()
-    manta_vcf           = Channel.empty()
-    mpileup_vcf         = Channel.empty()
-    strelka_vcf         = Channel.empty()
-    tiddit_vcf          = Channel.empty()
+    vcf_deepvariant     = Channel.empty()
+    vcf_freebayes       = Channel.empty()
+    vcf_haplotypecaller = Channel.empty()
+    vcf_manta           = Channel.empty()
+    vcf_mpileup         = Channel.empty()
+    vcf_strelka         = Channel.empty()
+    vcf_tiddit          = Channel.empty()
```

Comment:
I think we are getting to a point where we can remove this and manage the calls to VC with `when`, but maybe in another of refactoring, or we can have a new discussion, if this is still what we want